### PR TITLE
Remove some builder functions usage

### PR DESCRIPTION
**Pull request by olpipi**
- ngraph::builder::makeInputLayer function is being deleted from openvino repo. Exclude from contrib repo as well 
- ngraph::builder::makeParameter and ngraph::builder::makeDynamicParameter function is being deleted from openvino repo. Exclude from contrib repo as well

### CODE REVIEW

**Comment by olpipi** [comment]
Please do not merge it until all builder calls are deleted

**Comment by olpipi** [comment]
Please review the PR

File: modules/nvidia_plugin/tests/functional/shared_tests_instances/single_layer_tests/convolution_backprop_data.cpp [hunk 1]
```diff
@@ -92,12 +92,17 @@ class ConvolutionBackpropDataExtendedLayerTest
...
         auto shape = in.get_shape();
         std::vector<size_t> filterWeightsShape = {shape[1], numOutChannels};
         filterWeightsShape.insert(filterWeightsShape.end(), filterSize.begin(), filterSize.end());
-        auto filterWeightsNode =
-            ngraph::builder::makeConstant(type, filterWeightsShape, filterWeights, randomFilterWeights);
+        std::shared_ptr<ov::op::v0::Constant> filterWeightsNode;
+        if (filterWeights.empty()) {
+                ov::Tensor random_tensor(type, filterWeightsShape);
+                ov::test::utils::fill_tensor_random(random_tensor);
+                auto constant = std::make_shared<ov::op::v0::Constant>(random_tensor);
```
Comment (nkogteva) [inline modules/nvidia_plugin/tests/functional/shared_tests_instances/single_layer_tests/convolution_backprop_data.cpp:102]: Where `constant` is used? Should it be `filterWeightsNode`?

Comment (olpipi) [reply modules/nvidia_plugin/tests/functional/shared_tests_instances/single_layer_tests/convolution_backprop_data.cpp]: My fault. Fixed

**Review by nkogteva — APPROVED** [review]

**Pull request merged**